app/helpers/application_helper.rb

```diff
@@ -4,6 +4,7 @@ module ApplicationHelper
   include_concern 'FormTags'
   include Sandbox
   include CompressedIds
+  include TextualHelper
```

Comment:
This should be changed to reflect the new name of `TextualSummaryHelper` that you and @Fryguy decided on.